doc/man3/PKCS12_SAFEBAG_get0_attrs.pod

```diff
@@ -0,0 +1,44 @@
+=pod
+
+=head1 NAME
+
+PKCS12_SAFEBAG_get0_attrs,PKCS12_get_attr_gen - Retrieve attributes from a PKCS#12 safeBag
+
+=head1 SYNOPSIS
+
+ #include <openssl/pkcs12.h>
+
+ const STACK_OF(X509_ATTRIBUTE) *PKCS12_SAFEBAG_get0_attrs(const PKCS12_SAFEBAG *bag);
+
+ ASN1_TYPE *PKCS12_get_attr_gen(const STACK_OF(X509_ATTRIBUTE) *attrs,
+                                int attr_nid)
+
+=head1 DESCRIPTION
+
+PKCS12_SAFEBAG_get0_attrs() retrieves the stack of B<X509_ATTRIBUTE>s from a PKCS#12 safeBag.
+B<bag> is the B<PKCS12_SAFEBAG> retrieve the attributes from.
+
+PKCS12_get_attr_gen() retrieves an attribute by NID from a stack of B<X509_ATTRIBUTE>s.
+B<attr_nid> is the NID for the attribute to retrieve.
+
+=head1 RETURN VALUES
+
+PKCS12_SAFEBAG_get0_attrs() returns the stack of B<X509_ATTRIBUTE>s from a PKCS#12 safeBag, which could be empty.
+
+PKCS12_get_attr_gen() returns an ASN1_TYPE object containing the attribute, or NULL if the attribute was not present or an error occurred.
+PKCS12_get_attr_gen() does not allocate a new attribute. The returned attribute is still owned by the B<PKCS12_SAFEBAG> in which it resides.
+
+=head1 SEE ALSO
+
+L<PKCS12_add_friendlyname_asc(3)>,
+
+=head1 COPYRIGHT
+
+Copyright 2002-2019 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
This should be 2019 only, but it seems `make update` could fix this as well.

Reply:
I don't see how any update would make it 2019 only...

Reply:
Ahh, I recall there must be some scripts that can update this date 'up to date' when releasing a new version ;-)

Reply:
This was some copypasta I left in. Fixed now.

Reply:
"copypasta" - Nice pun. Reminds me of "spaghetti code". I'll remember that one :laughing: